borg/archiver.py

```diff
@@ -1,9 +1,10 @@
 from binascii import hexlify, unhexlify
 from datetime import datetime
-from hashlib import sha256
 from operator import attrgetter
+from string import Formatter
```

Comment:
Yep, remnant of moving the ItemFormatter class from archiver.Archiver to helpers.